style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: playfair Display, sans-serif;
+}
+
+body {
+  background-image: url("img/image_geometry_1.svg");
+  background-repeat: no-repeat;
+}
+
+.section-1 {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY css you can combine duplicate properties. 
You can create class `.d-flex { display: flex}` and use this class in html file.

- For more information about DRY principles you can check this [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/)